### ISSUE TITLE
Bug with type bubbles

### ISSUE THREAD

**Issue by randanderson** [mamezito/FramerMessengerChatBot#1]
When I set chatBot.Message to type:"bubbles", no bubbles render. Instead, I get an error that says

`Can't find variable: customTabBar`

I am using the latest Framer library, v101.

Thanks in advance for any guidance.

**Comment by mamezito-zz** [comment]
I don't think it will work. Too much stuff has been changed in Framer since I started that, but I will check
